webcompat/templates/user-activity/reported-by-user.html

```diff
@@ -1,11 +1,11 @@
-<div class="wc-UIContent" id="user-reported-issues">
-  <div class="wc-ListIssue-subtitle">
-    <h2 class="wc-Title wc-Title--l">Issues reported by {{user}}</h2>
-  </div>
-  <div class="wc-ListIssue-pagination">
-    {% include "web_modules/pagination.html" %}
-  </div>
-  <div id="my-issues">
-    {% include "web_modules/issue-list.jst" %}
-  </div>
-</div>
+<article class="grid">
+	<div class="grid-row" id="user-reported-issues">
```

Comment:
This file and the next is using tabs, and the previous one is using spaces for whitespace. Can we make it so all are using whitespace (2 spaces seems to be our convention)?